src/AppBundle/Controller/BlogController.php

```diff
@@ -94,6 +94,14 @@ public function commenterAction($id, Request $request)
         $comment->setPost($post);
 
         if ($form->handleRequest($request)->isValid()) {
+
+            /**
+             * "$this->getUser()" est une méthode permettant de récupérer l'utilisateur courant
+             * Cette méthode renvoie "null" si l'utilisateur n'est pas connecté
+             * @see http://api.symfony.com/2.6/Symfony/Bundle/FrameworkBundle/Controller/Controller.html#method_getUser
+             */
+            $comment->setAuteur($this->getUser());
```

Comment:
`$this->getUser()` dans un contrôleur permet de récupérer l'utilisateur connecté, et si l'utilisateur n'est pas connecté, cette fonction renvoie `null`